package.json

```diff
@@ -68,6 +68,7 @@
     "msw": "^0.39.1",
     "pm2": "^5.2.0",
     "prettier": "^2.5.1",
+    "prettier-plugin-tailwindcss": "^0.1.8",
```

Comment:
Is this still an issue with tailwind 3? https://github.com/tqwewe/prettier-plugin-tailwind/issues/29

Reply:
This is the official plugin by the Tailwind team.

`prettier-plugin-tailwind` (without the `css` at the end), is deprecated now
https://github.com/tqwewe/prettier-plugin-tailwind/issues/46#issuecomment-1020509621

Reply:
Nice! Thanks!